.appveyor.yml

```diff
@@ -20,18 +20,22 @@ environment:
   - db: mssql
     driver: sqlsrv
     db_version: sql2008r2sp2
+    coverage: yes
     php: 7.3
   - db: mssql
     driver: sqlsrv
     db_version: sql2012sp1
+    coverage: yes
     php: 7.3
   - db: mssql
     driver: sqlsrv
     db_version: sql2017
+    coverage: no
```

Comment:
This is only needed if Xdebug is used. Since we're switching to PCOV, would it make sense to collect code coverage just always on AppVeyor?

The configuration of Travis could be changed in a subsequent PR.